pkg/storage/fs/cephfs/cephfs.go

```diff
@@ -22,21 +22,29 @@
 package cephfs
 
 import (
+	"bytes"
 	"context"
+	b64 "encoding/base64"
+	"encoding/binary"
+	"encoding/json"
 	"fmt"
+	"hash"
+	"hash/fnv"
 	"io"
 	"net/url"
 	"os"
 	"path/filepath"
 	"strconv"
 	"strings"
+	"time"
 
-	cephfs2 "github.com/ceph/go-ceph/cephfs"
+	goceph "github.com/ceph/go-ceph/cephfs"
```

Comment:
Yeah, at least it's in its own commit. I did it before the other changes because I found it very confusing to have the package alias so similar to the package name. Not worth now to extract it in a separate PR...

Reply:
In any case, I totally agree with the name change!